manageiq-providers-autosde.gemspec

```
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "autosde_openapi_client", "=1.0.3"
```

Prefer

```suggestion
  spec.add_dependency "autosde_openapi_client", "~> 1.0"
```

This will allow patch-level and minor-level updates to automatically be pulled in, so you don't have to constantly be keeping this number up to date.